### ISSUE TITLE
Ends my program

### ISSUE THREAD

**Issue by Norway174** [ldilley/minestat#10]
https://github.com/ldilley/minestat/blob/ea221ae6bcef6b2945564552755c8d5db8cdc37c/JavaScript/minestat.js#L75

`process.exit()` also stops my node.js script.
And if I comment it out, it posts the same message twice if the server is offline.

Maybe I'm doing something wrong here?

**Comment by Norway174** [comment]
I fixed it by commenting out `callback();` at line 73 and `process.exit();` at line  #75.

```
    client.on('timeout', () =>
    {
      //callback();
      client.end();
      //process.exit();
    });
```

**Comment by ldilley** [comment]
Can you provide more details on the condition(s) that induce this symptom? I am unable to reproduce this behavior when I try with an invalid address (lookup error), use an address and port that actively refuse the connection (connection refused), or an address and port combination that time out.

Thank you for any additional information you might be able to provide.

**Comment by Norway174** [comment]
Hey. Sorry for my slow delay.

I've been hard at work on my Discord bot, where I am using Minestat as a part of a command to check the status of our minecraft server.

Here is the bot, if you wanna give it a look, or test it out.
https://github.com/Norway174/Peanut-DiscordBot

I hope you don't mind me including minestat in my repo. I assumed from you license it would be fine if I uploaded the modified version.
I will of course give credit to you.


Anyway,
The problem. As I pointed out. The `process.exit();` also exits my bot.
However, with the changes I made to your minestat.js, it works just fine. But it feels like it's taking a bit longer than the timeout I have set it to. It's not a huge problem. I haven't timed it yet. Been working on other parts of the bot.

It just takes a little bit longer to get the server status.
